crm_claim_type/models/crm_claim_stage.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 URSA Information Systems
```

Comment:
order inverted here?

Reply:
These lines of copyright should be ordered by year ?

Reply:
I have ordered the libes by year :)